src/sentry/options/store.py

```diff
@@ -203,8 +203,13 @@ def set_store(self, key, value):
 
     def set_cache(self, key, value):
         cache_key = key.cache_key
+
         if key.ttl > 0:
             self._local_cache[cache_key] = _make_cache_value(key, value)
+
+        if self.cache is None:
+            return
```

Comment:
I feel like we should raise on mutation calls instead of returning values. I'd rather just disallow any mutations until the store is fully initialized.